client/src/components/MultiJurisdictionAudit/Progress/ProgressMap.tsx

```diff
@@ -0,0 +1,367 @@
+import React, { useState, useRef, useEffect, useCallback } from 'react'
+import styled from 'styled-components'
+import { select, json, geoPath, geoAlbers } from 'd3'
+import { feature } from 'topojson-client'
+// topojson-specification is defined in package.json but throwing linting error here
+// eslint-disable-next-line import/no-unresolved
+import { Topology } from 'topojson-specification'
+import { Feature } from 'geojson'
+import { Colors, Spinner } from '@blueprintjs/core'
+import labelValueStates from '../AASetup/Settings/states'
+import { JurisdictionRoundStatus, IJurisdiction } from '../useJurisdictions'
+import { FileProcessingStatus, IFileInfo } from '../useCSV'
+
+interface IProps {
+  stateName?: string | null
+  jurisdictions: IJurisdiction[]
+  isRoundStarted: boolean
+  auditType:
+    | 'BALLOT_POLLING'
+    | 'BATCH_COMPARISON'
+    | 'BALLOT_COMPARISON'
+    | 'HYBRID'
+}
+
+const MapWrapper = styled.div`
+  position: relative;
+`
+
+const SVGMap = styled.svg`
+  margin-bottom: 30px;
+  .outline {
+    fill: none;
+    stroke: ${Colors.BLACK};
+    stroke-width: 0.5px;
+  }
+  .mesh {
+    fill: none;
+    stroke-linejoin: round;
+  }
+  path {
+    fill: none;
+    stroke: none;
+    stroke-width: 0.5px;
+  }
+  #single-state {
+    stroke: ${Colors.BLACK};
+  }
+  .county {
+    cursor: pointer;
```

Comment:
having the pointer made me think i could click on the map. i think leaving the default might be better